thirdeye/thirdeye-frontend/app/pods/home/index/controller.js

```diff
@@ -26,20 +26,19 @@ export default Controller.extend({
 
   init() {
     this._super(...arguments);
-    get(this, 'anomalyResponseObj').push({
+    // Add ALL option to copy of global anomaly response object
+    const anomalyResponseObj = _.cloneDeep(anomalyUtil.anomalyResponseObj);
+    anomalyResponseObj.push({
       name: 'All Resolutions',
       value: 'ALL',
       status: 'All Resolutions'
     });
+    setProperties(this, {
+      anomalyResponseObj,
```

Comment:
i think we should rename this to something relate to `Filter By` since it was meant for that and not to be confused with the actual anomaly response list. If that makes sense.